openupgradelib/openupgrade_merge_records.py

```diff
@@ -365,6 +366,10 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
         if field.type in ('char', 'text', 'html'):
             if not op:
                 op = 'other' if field.type == 'char' else 'merge'
+            if op == 'first_not_null':
+                _list = filter(lambda x: x, _list)
+                if _list:
+                    vals[field.name] = _list[0]
             if op == 'merge':
```

Comment:
Just for a little code performance optimization
```suggestion
            elif op == 'merge':
```

Reply:
done